tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -154,10 +154,15 @@ impl Default for Format<Full, SystemTime> {
 impl<T> Format<Json, T> {
     /// Use the full JSON format with event metadata flattened.
     ///
+    /// # Example Output
+    ///
+    /// ```ignore,json
+    /// {"timestamp":"Feb 20 11:28:15.096","level":"INFO","target":"mycrate", "message":"some message", "key": "value"}
+    /// ```
```

Comment:
similar to my comment in above, it might be worth also having this comment on the `LayerBuilder` and `SubscriberBuilder` `flatten_event` methods, or link to here? since that's the main way users will actually configure the JSON formatter.